### PR TITLE
Refactor flex-box to grid (update)

### LINKED ISSUE #136: Officer image sizing inconsistencies
The sizing of the ACM officer images are inconsistent; some appear smaller than others even when set to the same width and height in CSS.

### PR DESCRIPTION
**Pull request by anhduy1202**
Original: #197.

### CODE REVIEW

File: src/lib/components/about/officer-profile-list.svelte [hunk 1]
```diff
@@ -62,9 +62,10 @@
   }
```
Comment (EthanThatOneKid) [inline src/lib/components/about/officer-profile-list.svelte:63]: We could make sure the parent has a desired width:

```suggestion
.container {
  width: 100%;
  max-width: 1200px;
}
```

File: src/lib/components/about/officer-profile-list.svelte [hunk 2]
```diff
@@ -62,9 +62,10 @@
...
   .officer-profile-list {
-    display: flex;
-    flex-flow: row wrap;
-    max-width: 1400px;
-    justify-content: space-evenly;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+    justify-content: center;
+    align-items: center;
+    max-width: 1200px;
```
Comment (EthanThatOneKid) [inline src/lib/components/about/officer-profile-list.svelte]: Remove `justify-content` and `max-width` from this style declaration.

```suggestion
    align-items: center;
```

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
This is a good start; thank you! When I tried resizing the screen for mobile, I noticed that the grid was not resizing properly. I left some comments that hopefully helps resolve that small bug.

File: src/lib/components/about/officer-profile-list.svelte (hunk 1 above)
Comment (anhduy1202) [reply src/lib/components/about/officer-profile-list.svelte:63]: done!

File: src/lib/components/about/officer-profile-list.svelte (hunk 2 above)
Comment (anhduy1202) [reply src/lib/components/about/officer-profile-list.svelte]: done!

**Review by EthanThatOneKid — APPROVED** [review]
LGTM! 😎

**Pull request merged**